libs/python.json

```diff
@@ -122,10 +124,13 @@
   {"id":"py.rest-framework","imports":["rest_framework"],"name":"rest_framework","repo":"encode/django-rest-framework","tags":["django-rest"],"tech":["python-web"],"status":"model-ready"},
   {"id":"py.sanic","imports":["sanic"],"name":"sanic","repo":"channelcat/sanic","tags":["micro-framework"],"tech":["python-web"],"status":"model-ready"},
   {"id":"py.tornado","imports":["tornado"],"name":"tornado","repo":"tornadoweb/tornado","tags":["async-framework"],"tech":["python-web"],"status":"model-ready"},
+  {"id":"py.twisted","imports":["twisted."],"name":"Twisted","repo":"twisted/twisted","tags":["networking-engine"],"tech":["python-web"],"status":"awaiting-model"},
```

Comment:
I don't think you should be adding a dot to imports. Someone can just import twisted, and it will be perfectly fine.

Reply:
ok, you have a point